### PR TITLE
added proverb implementation in Go

### PR DESCRIPTION
**Pull request by sunilgopinath**
Closed previous PR and opened a new one after comments

### CODE REVIEW

File: config.json [hunk 1]
```diff
@@ -1177,6 +1177,18 @@
...
+    {
+      "core": false,
+      "difficulty": 4,
+      "slug": "proverb",
+      "topics": [
+        "arrays",
+        "loops",
+        "strings"
+      ],
+      "unlocked_by": "two-fer",
```
Comment (tleen) [inline config.json]: Good job here @sunilgopinath the only suggestion I would have is for the exercise unlock, seems like it falls into the general category of repeat a string with slight positional variations, which sits nicely in this subtree under [twelve-days](../tree/master/exercises/twelve-days):

```text
├─ twelve-days
│  ├─ house
│  ├─ food-chain
│  └─ beer-song
│
```

We can merge this as is and patch that change later. It is related to the next-gen exercism which is a later concern. I imagine we may revisit all the ordering before that anyhow.

Comment (sunilgopinath) [reply config.json]: That makes sense. I can update the ordering on another PR. I used the java repo as an example as I wasn't sure what to put there.

Comment (tleen) [reply config.json]: Yeah it can be tricky and sort of language dependent, your approach to guesstimating is as good as any :smile:.

**Comment by tleen** [comment]
Thanks for doing this, it looks great.

**Pull request merged**